strings/strings.json

```diff
@@ -3560,7 +3560,7 @@
   "space_red_biodome_desc_carn": "Raise cattle to produce food",
   "space_red_asphodel_desc": "Cultivate asphodel flowers to attract mediocre souls",
   "space_red_biodome_effect": "+%0 %1 Production per colonist",
-  "space_red_biodome_effect2": "+%0 Average Citizens per Living Quarter",
+  "space_red_biodome_effect2": "+%0 Average Colonist per Living Quarter",
```

Comment:
This would make it wrong. Citizens is the correct term here.

Reply:
Oh, in the key 'wiki_tech_hydroponics' is said that it is Colonists, and when I look in the code this appeared to be true. But looking better now, I see that it is really Citizens. I will fix it.